example/README.md

```diff
@@ -18,6 +18,21 @@ A number of use cases is presented in order to introduce different data structur
 	pip install -U wapi-python
 	```
 
+
+## Installation of Pytz library
+
+	```command
+	pip install pytz
+	```
```

Comment:
Pytz is listed as dependency in line 11. What is the motivation to have a special subsection for installing pytz here?

@hieuhc @stigalvestad You have been working on the 2038-issue. Is there a necessity to mention something about that issue and pytz in the docs?